annotations/views.py

```diff
@@ -268,12 +268,14 @@ def list_user(request):
     sort_dict = {"user_name":"username", "name":"full_name",
```

Comment:
Sorry if we talked about this before: why do we need a separate set of names for fields? Can't we just use the field names on the model?